### ISSUE TITLE
How were the synthetic scribbles generated during training?

### ISSUE THREAD

**Issue by aL3x-O-o-Hung** [halleewong/ScribblePrompt#8]
So I generated some fake scribbles on our dataset using the ground truth labels but the pretrained model cannot predict anything meaningful while using the generate clicks and bounding boxes are fine. I am wondering how the scribbles are generated and how the generated scribbles can shift the input distribution so much that it completely confuse the network.

**Comment by halleewong** [comment]
Can you show some examples?

The scribbles were generated using the code in [`scribbleprompt/scribbles.py`](https://github.com/halleewong/ScribblePrompt/blob/main/scribbleprompt/scribbles.py) during training.

**Comment by aL3x-O-o-Hung** [comment]
Hi! 

Thanks for the reply. Please find these examples attached. 


![microUS_test_01_positive_slice_11](https://github.com/user-attachments/assets/29c2665b-b35f-4468-af61-d1834f26a86d)
![microUS_test_01_positive_slice_16](https://github.com/user-attachments/assets/b787e072-4be7-4020-a483-823dc4bd310c)

**Comment by halleewong** [comment]
Those scribbles look quite thick, relative to the size of the images. The random deformations in our scribble generation code vary the scribble thickness a bit the training scribbles are rarely as wide and as harp as these examples. I suspect the predictions will be much better with thinner scribbles and/or lightly blurring the scribble mask.
